client/views/msg_condensed.tpl

```diff
@@ -1,7 +1,7 @@
 <div class="msg {{type}} closed" data-time="{{time}}">
 	<span class="time hide-text">{{tz time}}</span>
-	<span class="from"><span class="expander"></span></span>
+	<span class="from"></span>
 	<span class="content">
-		<span class="text condensed-msg"></span>
+		<span class="condensed-text"></span>
```

Comment:
Removed `text` class here to differentiate normal messages from condensed messages. See above for a further explanation.